pkg/sql/logictest/testdata/logic_test/jobs

```diff
@@ -270,8 +270,6 @@ SELECT count(*) FROM [SHOW AUTOMATIC JOBS] WHERE job_type = 'POLL JOBS STATS' AN
 
 subtest control_job_priv
 
-skip 125212
-
 user testuser
 
 statement ok
```

Comment:
While this subtest didn't fail, perhaps add a row here too and any other subtest?

Reply:
not sure if it matters for this subtest, which is just asserting privileges work, but will add it here.

Reply:
right, but the test expects to hit the `user testuser does not have privileges for job` error. this privilege check occurs before actually trying to pause the job. (just being pedantic; i agree with the point that it's better to avoid that brittle assumption in this test.)

Reply:
ah, i see. thanks for the additional clarification!